<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeBool</class>
 <widget class="QDialog" name="ChangeBool">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changing</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="trueButton">
     <property name="text">
      <string>True</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="falseButton">
     <property name="text">
      <string>False</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
